src/services/services.ts

```diff
@@ -2365,6 +2369,40 @@ module ts {
                 return result;
             }
 
+            function isCompletionListBuilder(previousToken: Node): boolean {
+                if (previousToken) {
+                    var containingNodeKind = previousToken.parent.kind;
+                    switch (previousToken.kind) {
+                        case SyntaxKind.CommaToken:
+                            return containingNodeKind === SyntaxKind.CallExpression
+                                || containingNodeKind === SyntaxKind.Constructor;
+                        case SyntaxKind.OpenParenToken:
+                            return containingNodeKind === SyntaxKind.CallExpression
+                                || containingNodeKind === SyntaxKind.Constructor;
+                        case SyntaxKind.ModuleKeyword:
+                            return true;
+                        case SyntaxKind.DotToken:
+                            return containingNodeKind === SyntaxKind.ModuleDeclaration;
+                        case SyntaxKind.OpenBraceToken:
+                            return containingNodeKind === SyntaxKind.ClassDeclaration;

```

Comment:
I don't think there is anything meaningful we can show in the completionlist in that case, so we don't show one.